Content.Shared/_Starlight/Silicons/StationAi/SharedStationAiSystem.Linking.cs

```diff
@@ -0,0 +1,45 @@
+using Content.Shared.DeviceLinking.Events;
+using Content.Shared.Silicons.Laws.Components;
+using Content.Shared.Silicons.StationAi;
+
+namespace Content.Shared.Silicons.StationAi;
+
+public abstract partial class SharedStationAiSystem
+{
+    // Handles device linking
+
+    private void InitializeLinking()
+    {
+        SubscribeLocalEvent<StationAiCoreComponent, NewLinkEvent>(OnNewLink);
+        SubscribeLocalEvent<StationAiCoreComponent, PortDisconnectedEvent>(OnPortDisconnected);
+    }
+    
+    private void OnNewLink(Entity<StationAiCoreComponent> ent, ref NewLinkEvent args)
+    {
+        if (!TryComp<SiliconLawUpdaterComponent>(args.Sink, out var lawUpdater))
+            return;
+
+        ent.Comp.LawConsole = GetNetEntity(args.Sink);
+
+        lawUpdater.Core = GetNetEntity(ent.Owner);
```

Comment:
```suggestion
        lawUpdater.Core = ent.Owner;
```

NetEntity can't be DataField